### PR TITLE
fix: add li directly with list style

### PR DESCRIPTION
**Pull request by stefanoScalzo**
## Related Issue
Closes SAP/fundamental-styles #768

## Description
This pull request is to help target product switch that enables drag and drop. The drag and drop features changes the structure of the DOM, therefore, we need to explicitly target the li to remove bullets
## Screenshots
###Before
Screen Shot 2020-03-05 at 1 30 12 PM
###After
Screen Shot 2020-03-05 at 1 29 23 PM

### CODE REVIEW

File: src/product-switch.scss [hunk 1]
```diff
@@ -57,6 +57,10 @@ $block: #{$fd-namespace}-product-switch;
     }
 
     width: 47rem;
+
+    > li {
+      list-style-type: none;
```
Comment (InnaAtanasova) [inline src/product-switch.scss]: Could you add this rule to `&__item` too?

**Comment by stefanoScalzo** [comment]
> Can you maintain the issue attributes and move the rule to the class?

What do you mean by issue attributes?

File: src/product-switch.scss (hunk 1 above)
Comment (stefanoScalzo) [reply src/product-switch.scss]: If we do this why don't we just target all li under list and make them list-style-type:none

**Comment by stefanoScalzo** [comment]
Needs to be done on ngx side

**Pull request closed without merging**